### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by Nurjadein**
## Pull Request Checklist

- [ ] Does this PR includes changes from server side such as workflow, gateway, etc.?
- [x] Have you removed any unnecessary console.log?
- [x] Have you checked all variables, class names, function names, components, file names etc comply with our coding standards? 
- [x] Have you checked any spelling errors/issues?

### CODE REVIEW

File: src/pages/employees/assign-working-shift/UpsertWorkingScheduleContent.vue [hunk 1]
```diff
@@ -219,17 +241,22 @@ export default class UpsertWorkingScheduleContent extends Vue.with(Props) {
...
+      this.showSimulatedReturnData = false;
     } else {
       this.convertDays();
-      this.selectedSched = this.selectedWorkSchedule as never;
+      this.selectedSched = { ...this.selectedWorkSchedule };
       this.showValues = true;
+      this.showSimulatedReturnData = true;
     }
+    this.visible = false;
+    console.log(this.selectedSched);
```
Comment (mosmdansalan) [inline src/pages/employees/assign-working-shift/UpsertWorkingScheduleContent.vue:252]: please remove this console.log

File: src/pages/common/ProfilePage.vue [hunk 2]
```diff
@@ -467,26 +478,55 @@ export default class EmployeeProfile extends Vue {
...
-    this.showDialog({ show: true, event: 'updateProfile' });
+  get startTime() {
+    return ParsingUtil.formatTime(this.selectedWorkSchedule.startTime);
   }
-  showDialog(val: { show: boolean; event: string }) {
-    this.dialogTitle = val.event;
-    this.isShowDialog = val.show;
+
+  get endTime() {
+    const v = ParsingUtil.formatTime(this.selectedWorkSchedule.endTime);
```
Comment (mosmdansalan) [inline src/pages/common/ProfilePage.vue:504]: you may remove this variable `v`. It's not used.

Comment (Nurjadein) [reply src/pages/common/ProfilePage.vue]: yes sir

File: src/pages/employees/assign-working-shift/UpsertWorkingScheduleContent.vue (hunk 1 above)
Comment (Nurjadein) [reply src/pages/employees/assign-working-shift/UpsertWorkingScheduleContent.vue]: yes sir

**Review by mosmdansalan — APPROVED** [review]

**Pull request merged**